llvm/lib/Target/SPIRV/SPIRVEmitIntrinsics.cpp

```diff
@@ -262,6 +264,33 @@ class SPIRVEmitIntrinsics
   }
 };
 
+struct FPFastMathDefaultInfo {
```

Comment:
This is an exact copy of the FPFastMathDefaultInfo struct in the ModuleAnalysis? Why not reuse it?